example.py

```diff
@@ -1,10 +1,16 @@
 import sys
-from recasepunc import CasePuncPredictor
+from recasepunc import CasePuncPredictor, \
+    WordpieceTokenizer  # WordpieceTokenizer is necessary for flavors other than fr
 
-predictor = CasePuncPredictor('fr-txt.large.19000')
+predictor = CasePuncPredictor('it.22000')
```

Comment:
Maybe need something more language independent

Reply:
Yes, I agree.

We could read from the **LANG** environment variable (also used in the *README*):

```
predictor = CasePuncPredictor(join('checkpoints', getenv('LANG')))
```